src/Sarif.Converters/FortifyFprConverter.cs

```diff
@@ -243,10 +243,16 @@ private Run CreateSarifRun()
         {
             if (!string.IsNullOrWhiteSpace(originalUriBasePath))
             {
-                if (originalUriBasePath.StartsWith("/") &&
-                     platform == "Linux")
+                if (originalUriBasePath.StartsWith("/"))
                 {
-                    originalUriBasePath = "file://" + originalUriBasePath;
+                    if (platform == "Linux")
```

Comment:
>platform [](http://example.com/codeflow?start=24&length=8)

Is this all working for mac vs. linux vs. windows?

Reply:
changed to check for windows instead